### PR TITLE
Adding I should see in command output

### PR DESCRIPTION
**Pull request by guillaumevoisin**

### CODE REVIEW

File: src/Knp/FriendlyContexts/Context/CommandContext.php [hunk 1]
```diff
@@ -141,6 +141,18 @@ public function commandOutputShouldBeLike(PyStringNode $string)
     }
 
     /**
+     * @Then I should see :string in the command output
+     */
+    public function iShouldSeeInTheOutput($string)
+    {
+        $commandOutput = $this->getRawCommandOutput();
+
+        if (strpos($commandOutput, $string) === false) {
```
Comment (vincentchalamon) [inline src/Knp/FriendlyContexts/Context/CommandContext.php]: Yoda style

Comment (vincentchalamon) [inline src/Knp/FriendlyContexts/Context/CommandContext.php]: Isn't it to strict to compare exact output? Can be embarrassing about spaces

**Comment by vincentchalamon** [comment]
If this works well, can you also propose your modification to https://github.com/KnpLabs/FriendlyContexts/pull/182?

**Comment by guillaumevoisin** [comment]
@vincentchalamon yes would be nice

**Comment by MLKiiwy** [comment]
👍  and yeah propose a PR on the real repo

**Pull request merged**